### DISCUSSION TITLE
Welcome to "Build a Frontend Framework" Discussions!

### DISCUSSION THREAD

**Discussion by angelsolaorbaiceta** [angelsolaorbaiceta/fe-fwk-book#193] [Ideas]
## Build a Frontend Framework — Discussions

  You can use Discussions as a place to discuss your ideas. Here you can:
  * Share ideas about improvements to the framework.
  * Propose features that'd make the framework more useful.
  * Engage with other readers of ["Build a Frontend Framework (from scratch)"](http://mng.bz/aM2o).
  * Learn from others.

I ask you to be respectful and listen to other's ideas and point of views with curiosity and an open mind.

Please take the time to write a good description of your proposal. Get yourself familiar with [Github's Markdown syntax](https://docs.github.com/es/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax), include code snippets and illustrations.

**Comment by patentfox** [comment]
I'm enjoying your book a lot at the moment. Just curious, what tool do you use to generate your diagrams? They look very clean. If you have a template, I'd love to take a look.

Thanks!
